src/_fastmath.c

```diff
@@ -1571,6 +1585,7 @@ getStrongPrime (PyObject *self, PyObject *args, PyObject *kwargs)
 	mpz_clear (p[0]);
 	mpz_clear (y[1]);
 	mpz_clear (y[0]);
+	mpf_clear (tmp_bound);
```

Comment:
Just to head-off comments, this is not a typo. `tmp_bound` is allocated with a different function called `mpf_init` (which is from a slightly different library than `mpz_init`), so it needs to be cleared with `mpf_clear` (not `mpz_clear` like we've been using everywhere else).